### DISCUSSION TITLE
Do the versions of react-dasiyui correspond to daisyui?

### DISCUSSION THREAD

**Discussion by arthtyagi** [daisyui/react-daisyui#172] [Q&A] [answered]
[DaisyUI](https://github.com/saadeghi/daisyui) is on version 2.19.0 whereas [`react-daisyui`](https://github.com/daisyui/react-daisyui/) seems to be on version 2.0.6.

So, do the versions of [`react-daisyui`](https://github.com/daisyui/react-daisyui/) correspond to the versions of [DaisyUI](https://github.com/saadeghi/daisyui) implying [`react-daisyui`](https://github.com/daisyui/react-daisyui/) is slightly behind due to the port?

**Comment by benjitrosch** [comment, marked as answer]
Hi @arthtyagi,

No, the version numbers are not meant to match up. We are currently up to date with `daisyUI`. We follow our own versioning as we release updates, fixes, and changes, and we are on an independent release schedule from daisyUI.

**Comment by MrChrisRodriguez** [comment]
I was wondering this same thing. 

I found it confusing that they don't match, mainly because the version numbers are so similar to each other (2.5 vs 2.43 currently). My initial thought was: "maybe the DaisyUI website hasn't updated their version but react-daisyui is tracking the repo which is updated."

Has any consideration been given to keeping the version numbers in sync? Some ideas include:
- keep major and minor version in sync, but patch can float with react changes
- keep major, minor, and patch in sync, and use the pre-release portion of the semver RFC to track react-specific fixes. For example, daisyui: 2.43.2 and react-daisyui: 2.43.2-2
